Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -978,6 +990,33 @@ public extension String {
     }
     #endif
 
+    #if canImport(Foundation)
+    /// SwifterSwift: Overload Swift's 'contains' operator for matching regex
+    ///
+    /// - Parameter lhs: String to check on regex.
+    /// - Parameter rhs: Regex to match against.
+    /// - Returns: `true` if there is at least one match for the regex in the string.
+    static func ~= (lhs: String, rhs: NSRegularExpression) -> Bool {
+        let range = NSRange(lhs.startIndex..<lhs.endIndex, in: lhs)
+        return rhs.firstMatch(in: lhs, range: range) != nil
+    }
+    #endif
+
+    #if canImport(Foundation)
+    /// SwifterSwift: Returns a new string in which all occurrences of a regex in a specified range of the receiver are replaced by the template.
+    /// - Parameter regex: Regex to replace.
+    /// - Parameter template: The template to replace the regex.
+    /// - Parameter searchRange: The range in the receiver in which to search.
+    /// - Returns: A new string in which all occurrences of regex in searchRange of the receiver are replaced by template.
+    func replacingOccurrences(
+        of regex: NSRegularExpression,
+        with template: String,
+        range searchRange: Range<String.Index>? = nil) -> String {
```

Comment:
You don't want `options` as a parameter of the function?
```suggestion
        with template: String,
        options: NSRegularExpression.MatchingOptions = [],
        range searchRange: Range<String.Index>? = nil) -> String {
```
If you use it make sure to update the CHANGELOG too.

Reply:
Good idea, I'll toss it in there. Sometimes it's easy to get stuck in a rut applying specific changes without looking at the big picture. 😄